### ISSUE TITLE
Apps not showing in main menu

### ISSUE THREAD

**Issue by nikclayton** [nikclayton/android-squeezer#71]
```
What steps will reproduce the problem?
1. go to main menu
2. apps entry not showing

What is the expected output? What do you see instead?
would like to control spotify third party plugin for example

What version of the product are you using? On what operating system?
V0.9.1 / Logitech Media Server Version : 7.7.2 - r33893 on ubuntu 12.04 distribution

Please provide any additional information below.
Like this app, especially vol UP/DOWN!
```

Original issue reported on code.google.com by `axiaxi35` on 2013-04-13 13:04:32

**Comment by nikclayton** [comment]
https://github.com/nikclayton/android-squeezer/wiki/Apps-support-in-Squeezer

**Comment by kaaholst** [comment]
Fixed in 2.0.0
